features/0592-indy-attachments/README.md

```diff
@@ -102,47 +158,48 @@ A complete [`request-credential` message from the Issue Credential protocol 2.0]
 
 ### credential format
 
-A concrete, issued Indy credential may be transmitted over many protocols, but is specifically expected as the final message in [Issuance Protocol 2.0](../0453-issue-credential-v2/README.md). The identifier for its format is `hlindy/cred@v2.0`, and its media (MIME) type is `application/indy-cl-anoncred2`.
+A concrete, issued Indy credential may be transmitted over many protocols, but is specifically expected as the final message in [Issuance Protocol 2.0](../0453-issue-credential-v2/README.md). The identifier for its format is `hlindy@v2.0` in the context of an Issue Credential message.
```

Comment:
Indy credentials definitely need their own format that distinguishes them from the format of a credential request or proposal.